server/scripts/import-tests/index.js

```diff
@@ -251,6 +253,38 @@ const readDirectoryGitInfo = directoryPath => {
     return { gitSha, gitMessage, gitCommitDate };
 };
 
+const importHarness = () => {
+    const sourceFolder = path.resolve(`${testsDirectory}/resources`);
+    const targetFolder = path.resolve('../', 'client/resources');
+    console.info('Clearing harness directory ...');
```

Comment:
```suggestion
    console.info(`Updating harness directory, ${targetFolder} ...`);
```

This and the following message suggestions are nitpicks and you can feel free to ignore the exact wording although I do feel better about seeing an explicit path being shared in operations like this.